index.ts

```diff
@@ -401,7 +403,7 @@ export class GCP extends CloudPlatform<
         // First must delete the ephemeral address on a host.
         // Then we can add the EIP(or static address in the case of GCP)
         try {
-            const deleteReq = await client.request({
+            const deleteReq:any = await client.request({
```

Comment:
looks like deleteReq is an object {data:{status:string},[key:string]:string}. Can use it to replace type 'any'.

Reply:
it is an object, but a pretty complicated one for the entire VM data, that changes a lot based on what vm etc. I could replace the return with the info I need instead.

Reply:
Oh sorry nevermind.  I misread. Yeah I should be able to make a type for deleteReq

Reply:
Added type for deleteReq
```    
export interface DeleteRequest {
        data: Data;
      }
      export interface Data {
        id: string;
        name: string;
        zone: string;
        operationType?: string;
        targetLink?: string;
        targetId?: string;
        status?: string;
        user?: string;
        progress?: number;
        insertTime?: string;
        startTime?: string;
        selfLink: string;
        kind?: string;
      }
      ```